x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -500,18 +498,62 @@ export class RemoteClusterForm extends Component {
   }
 
   renderErrors = () => {
-    const { areErrorsVisible } = this.state;
+    const {
+      areErrorsVisible,
+      fields: {
+        name,
+      },
+      fieldsErrors: {
+        name: errorClusterName,
+      },
+      localSeedErrors,
+      seedInput,
+    } = this.state;
+
     const hasErrors = this.hasErrors();
 
     if (!areErrorsVisible || !hasErrors) {
       return null;
     }
 
+    const errorExplanation = [];
+
+    if (errorClusterName) {
+      errorExplanation.push({
+        key: 'nameExplanation',
+        where: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputNameErrorsScreenReaderExplanation', {
+          defaultMessage: 'The input "Name". You have entered: {name}.',
+          values: { name }
+        }),
+        what: errorClusterName
+      });
+    }
+    if (localSeedErrors && localSeedErrors.length) {
+      errorExplanation.push({
+        key: 'seedExplanation',
+        where: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputSeedErrorsScreenReaderExplanation', {
+          defaultMessage: 'The "Seed Nodes" input. You have entered: {seedInput}.',
+          values: { seedInput }
+        }),
+        what: localSeedErrors.join(' ')
+      });
+    }
+
+    const messagesToBeRendered = (
+      errorExplanation.length && errorExplanation.map(errorMessage => (
+        <EuiScreenReaderOnly key={errorMessage.key}>
+          <p>
```

Comment:
Can we make a few changes to the errors?

1. Can we read aloud only the problematic field and the error with that field? This saves us the headache of having to figure out how to read the field value, which can be complex in different situations (e.g. reading out an empty field which is required in a way that makes sense).
2. Can we rename `errorExplanation` to `errorExplanations` (plural) to reflect its role as an array containing many items?
3. Can we add comments to explain why there are line breaks?

Here's what I would suggest:

```jsx
    const errorExplanations = [];

    if (errorClusterName) {
      errorExplanations.push({
        key: 'nameExplanation',
        field: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputNameErrorMessage', {
          defaultMessage: 'The "Name" field is invalid.',
        }),
        error: errorClusterName
      });
    }

    if (errorsSeeds) {
      errorExplanations.push({
        key: 'seedsExplanation',
        field: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputSeedsErrorMessage', {
          defaultMessage: 'The "Seed nodes" field is invalid.',
        }),
        error: errorsSeeds
      });
    }

    if (localSeedErrors && localSeedErrors.length) {
      errorExplanations.push({
        key: 'localSeedExplanation',
        field: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputLocalSeedErrorMessage', {
          defaultMessage: 'The "Seed nodes" field is invalid.',
        }),
        error: localSeedErrors.join(' '),
      });
    }

    const messagesToBeRendered = errorExplanations.length && (
      errorExplanations.map(({ key, field, error }) => (
        <EuiScreenReaderOnly key={key}>
          <section>
            <br /> {/* Without these break the paragraphs run together into single sentences. */}
            <p>{field}</p>
            <br />
            <p>{error}</p>
          </section>
        </EuiScreenReaderOnly>
      ))
    );
```